src/components/Calendar.tsx

```diff
@@ -0,0 +1,70 @@
+import AutoSizer from "react-virtualized-auto-sizer"
+import { useEffect, useRef } from "react"
+import { VariableSizeList } from "react-window"
+import { getWeeksInMonth } from "date-fns"
+
+import Month from "./CalendarMonth"
+import { getDateFromIndex, getIndexFromDate, MAX_DATE } from "../utils/dates"
+
+export const remToPixels = (rem: number): number =>
+  rem * parseFloat(getComputedStyle(document.documentElement).fontSize)
+
+export function calculateMonthHeight(index: number): number {
+  const marginBottom = 24
+  const monthTitle = remToPixels(1.6) + 12
+  const weekDayTitle = remToPixels(1.2) + 8
+
+  const date = getDateFromIndex(index)
+  const weeks = getWeeksInMonth(date)
+
+  const viewportWidth = Math.min(document.documentElement.clientWidth, 600)
+  const dayHeight = (viewportWidth - 8 - 8 - 6 * 4) / 7
+  const daysHeight = dayHeight * weeks
+  const gridGap = 8 * (weeks + 1)
+
+  return monthTitle + weekDayTitle + marginBottom + daysHeight + gridGap
+}
+
+function CalendarList({
```

Comment:
The reason that this component isn't covered by the Calendar snapshot is because shallow rendering only renders the top level component. You can export CalendarList and create a snapshot for it.

Reply:
Ah good point 👌🏼